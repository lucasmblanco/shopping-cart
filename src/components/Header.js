import React from 'react'
import logo from '../assets/logo-et.png';

export default function Header() {
  return (
    <div className=" h-32 lg:h-44 flex justify-center">
      <img className="m-10 object-cover" src={logo} alt="company-logo"></img>
    </div>
  );
}
