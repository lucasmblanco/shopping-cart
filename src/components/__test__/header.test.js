import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import Header from '../Header';

it('should render the component', () => {
  const { asFragment } = render(<Header />);

  expect(asFragment()).toMatchSnapshot();
});

it('should render image component', () => {
  render(<Header />);

  expect(screen.getByRole('img')).toBeInTheDocument();
});
