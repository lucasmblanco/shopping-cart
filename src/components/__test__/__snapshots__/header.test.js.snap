// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the component 1`] = `
<DocumentFragment>
  <div
    class="h-44 flex justify-center"
  >
    <img
      alt="company-logo"
      class="m-10 object-cover"
      src="logo-et.png"
    />
  </div>
</DocumentFragment>
`;
